examples/use_config_ifp/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+project(ignition-find_config VERSION 0.1.0)
+find_package(ignition-cmake2 REQUIRED)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/")
+set(ignition-find_config_DIR "${CMAKE_SOURCE_DIR}/")
+#ign_find_package(ignition-find_config) will search for Findignition-find_config.cmake before ignition-find_configConfig.cmake 
+ign_find_package(ignition-find_config)
+#After CONFIG arguemnt ign_find_package will skip MODULE mode and search for ignition-find_configConfig.cmake directly
+ign_find_package(ignition-find_config CONFIG)
+ign_configure_project()
+ign_configure_build(QUIT_IF_BUILD_ERRORS)
+ign_create_packages()
```

Comment:
It looks like these lines are making CI angry, I wonder if we should remove them. I see the following error output when I try to build this example locally:
```
CMake Error at /usr/local/share/cmake/ignition-cmake2/cmake2/IgnConfigureBuild.cmake:142 (add_subdirectory):
  add_subdirectory given source "src" which is not an existing directory.
Call Stack (most recent call first):
  CMakeLists.txt:11 (ign_configure_build)

...

-- Configuring incomplete, errors occurred!
```

I did notice that earlier in the output, `CONFIG` is working as expected, because I see the following messages:
```
-- The package was found using MODULE mode

...

-- The package was found using CONFIG mode
```

So, since this test is only meant to test `CONFIG` and not configuring the project/build, I'm leaning towards removing these lines since the `CONFIG` functionality that's being tested is working. What do you think, @j-rivero?

Reply:
Hey @adlarkin, adding an src folder with an empty CMakeLists.txt and an empty README.md seems to have solved the problem. Also, I have deleted all the unnecessary whitespaces.

Reply:
Great, thanks! Everything looks good to me :slightly_smiling_face: